design/workflow.go

```diff
@@ -193,6 +221,49 @@ var _ = Service("workflow", func() {
 			Response(CodeOK)
 		})
 	})
+
+	Method("listRuns", func() {
+		Description("List Workflow runs.")
+
+		Payload(func() {
+			Field(1, "workflowName", String, "Name of the Workflow to list runs from", func() {
+				Example("mlflow-sklearn-e2e")
+			})
+			Field(2, "codesetName", String, "Name of the codeset to list runs from", func() {
+				Example("mlflow-project-001")
+			})
+			Field(3, "codesetProject", String, "Name of the codeset project to list runs from", func() {
+				Example("workspace")
+			})
+			Field(4, "status", String, "status of the workflow runs to list", func() {
```

Comment:
"**S**tatus" ... to make it consistent with rest

Reply:
done